.github/workflows/ci.yml

```diff
@@ -41,6 +42,7 @@ jobs:
             gemfile: rails5.2
     env:
       BUNDLE_GEMFILE: gemfiles/${{ matrix.gemfile }}.gemfile
+      LEGACY_CONNECTION_HANDLING: "true"
```

Comment:
This is technically not needed since we have set it in the test helper, right?
https://github.com/zendesk/active_record_shards/pull/294/files#diff-c80b930c75386e46447f64c40e6844ae4c793fe0c4c1efef890ca758adc13a84R28

Reply:
Yep!